styles/nextjs/catppuccin.user.css

```
@@ -0,0 +1,256 @@
+    --ds-gray-100: @base;
+    --ds-gray-200: @surface0;
+    --ds-gray-300: @surface1;
+    --ds-gray-400: @surface0;
+    --ds-gray-700: @subtext0;
+    --ds-gray-1000: @text;
+    --ds-gray-900: @subtext0;
+    --ds-gray-800: @subtext1;
+
+    --ds-red-800: @red;
+    --ds-red-900: lighten(@red, 5%);
+
+    --ds-blue-300: @surface0;
+    --ds-blue-900: @accent-color;
+
+    /* Links vertical line */
+    .bg-blue-600 {
+      background-color: @accent-color;
+    }
+
+    /* App Router icon */
+    --ds-blue-700: @blue;
+
+    /* Pages Router icon */
+    --ds-purple-700: @mauve;
+
+    --ds-blue-400: {
+      color: @base;
+    };
+
+    --ds-purple-400: {
+      color: @base;
+    };
+
+    --tw-gradient-top: transparent;
+
+    a h3 {
+      color: @accent-color;
+    }
+
+    p a {
+      color: @accent-color;
+    }
+
+    ul li::marker {
+      color: @accent-color;
+    }
+
+    .text-blue-700 {
+      color: @accent-color;
+    }
+
+    svg {
+      --geist-stroke: var(--geist-background) !important;
+    }
+
+    button.navbar_search__dZT2b {
+      background-color: @mantle !important;
+    }
+
+    input::placeholder,
+    button[class*="navbar_search"] {
+      color: @text;
+    }
+
+    code {
+      color: @accent-color !important;
+    }
+
+    .shadow,
+    .shadow-md,
+    .shadow-sm {
+      box-shadow: none !important;
+    }
+
+    .code-block_wrapper__t6FCO {
+      border-color: 1px solid @surface0 !important;
+    }
+
+    .group:hover .dark\:group-hover\:bg-white {
+      background-color: @subtext1;
+    }
+
+    .group:hover .dark\:group-hover\:text-black {
+      color: @crust;
+    }
+
+    [data-docs-container] code:not(pre code),
+    [data-docs-table-of-contents] code:not(pre code) {
+      border-color: 1px solid @surface0 !important;
+      background-color: @mantle !important;
+    }
+
+    td a {
+      color: @accent-color;
+    }
+
+    .animated-optimizations_window__CjM85 {
+      background: @mantle !important;
+    }
+
+    .features_card__xYla9,
+    .foundation_root__wm0ez,
+    .foundation_card__v7VKB {
+      background: @base;
+    }
+
+    .foundation_cpu__ciXpm {
+      background: @mantle;
+    }
+
+    .header_header__zJOD0 {
+      background: @base;
+    }
+
+    .mute,
+    .subtitle {
+      color: @subtext0;
+    }
+
+    .mobile-menu_mobileMenu__xqbOP a.mobile-menu_selected__G55UL {
+      color: var(--geist-background);
+    }
+
+    .dark-theme .dark\:bg-black\/50 {
+      background-color: var(--geist-background);
+    }
+
+    .gradient,
+    .gradient-text {
+      --text-gradient: @text;
+    }
+
+    .bg-background-200,
+    .bg-vercel-100 {
+      background-color: @mantle;
+    }
+
+    .guides_book__j9vP8 {
+      background: @base !important;
+    }
+
+    .guides_bind__9COxI {
+      background: @overlay0 !important;
+    }
+
+    .bg-gray-50 {
+      background-color: @mantle;
+    }
+
+    .styled-scrollbar {
+      scrollbar-color: @accent-color transparent;
+    }
+
```

Code block syntax highlighting!
```suggestion

      .code-block_pre___OLfy {
        --shiki-token-punctuation: @overlay2;
        --shiki-token-constant: @peach;
        --shiki-token-string: @green;
        --shiki-token-string-expression: @green;
        --shiki-token-comment: @overlay0;
        --shiki-token-keyword: @mauve;
        --shiki-token-parameter: @maroon;
        --shiki-token-function: @blue;
      }

```